views/Data.js

```diff
@@ -244,103 +250,149 @@ export function useDataListener({
   }, []) // eslint-disable-line
 }
 
+export function useDataContext(context) {
+  return useContext(DataContexts[context])
```

Comment:
should we remove the default context from data contexts now?

so on line 72 of `Data.js`, we have:
```
let DataContexts = {
  default: React.createContext([]),
}
```
it could be
```
let DataContexts = {}
````

Then we could remove any reference to `context = 'default'` in Data.js (and probably Data.tools.js file too tbh) since the `useDataContext` hook will be dealing with it https://github.com/viewstools/morph/blob/7abb612d80ed5c9ffde0366c6db0dd4a56b1684a/views/Data.tools.js#L304.

What do you think?

Reply:
I may be missing something here: shouldn't the instantiation of the `default` context be left as it is i.e.
```
let DataContexts = {
  default: React.createContext([]),
}
```
as the `useDataContext` references it? or would you suggest to move the creation inside `useDataContext` if it doesn't already exist?

Reply:
I'd remove the default context in `Data.js` since we shouldn't need it. It should be enough ™️ with the warnings you get to know you should define the data provider otherwise it won't work. I'd leave it in `Data.tools.js` though because we use it as a fallback in dev mode.

Does that make sense?

Reply:
yes, sounds good